docs/diagrams/UiClassDiagram.puml

```diff
@@ -43,6 +45,7 @@ MainWindow *-down-> "1" EventContactDisplay
 MainWindow *-down-> "1" ContactListPanel
 MainWindow *-down-> "1" StatusBarFooter
 MainWindow *-down-> "1" TagListPanel
+MainWindow *-down-> "1" ReminderListPanel
```

Comment:
`MainWindow` does not contain the `ReminderListPanel`

Reply:
`EventContactDisplay` contains the `ReminderListPanel`

Reply:
Good catch too! Updated it in my following PR (#258)